<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="gs4" name="CWS Detailed Header Footer API" tests="13" failures="3" timestamp="19 Mar 2012 10:14:18 GMT" time="12.134" errors="0">
  <testcase name="testAbsoluteHead" time="3.027" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.RangeCheck(ArrayList.java:547)
	at java.util.ArrayList.get(ArrayList.java:322)
	at com.sonyericsson.automatedtests.page.HeaderFooterAPI.verifyAbsoluteHead(HeaderFooterAPI.java:28)
	at com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI.testAbsoluteHead(UsingHeaderFooterAPI.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:691)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:883)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1208)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:758)
	at org.testng.TestRunner.run(TestRunner.java:613)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:619)
]]>
    </failure>
  </testcase>
  <testcase name="testAbsoluteHeader" time="6.066" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI">
    <failure type="java.lang.AssertionError" message=" header contains something that is not relative expected:&amp;lt;true&amp;gt; but was:&amp;lt;false&amp;gt;">
      <![CDATA[java.lang.AssertionError:  header contains something that is not relative expected:<true> but was:<false>
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertTrue(Assert.java:37)
	at com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI.testAbsoluteHeader(UsingHeaderFooterAPI.java:48)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:691)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:883)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1208)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:758)
	at org.testng.TestRunner.run(TestRunner.java:613)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:619)
]]>
    </failure>
  </testcase>
  <testcase name="testPluginLocatorAbs" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginSearchBoxAbs" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginSearchBoxRel" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginSecLevelNavAbs" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginSecLevelNavJSAbs" time="0.0" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginSecLevelNavRel" time="0.0" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginShoppingCartAbs" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginShoppingCartJSAbs" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginShoppingCartJSRel" time="0.0010" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testPluginShoppingCartRel" time="0.0" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI"/>
  <testcase name="testRelativeHead" time="3.0" classname="com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI">
    <failure type="java.lang.IndexOutOfBoundsException" message="Index: 0, Size: 0">
      <![CDATA[java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
	at java.util.ArrayList.RangeCheck(ArrayList.java:547)
	at java.util.ArrayList.get(ArrayList.java:322)
	at com.sonyericsson.automatedtests.page.HeaderFooterAPI.verifyRelativeHead(HeaderFooterAPI.java:72)
	at com.sonyericsson.automatedtests.quick.UsingHeaderFooterAPI.testRelativeHead(UsingHeaderFooterAPI.java:40)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:691)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:883)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1208)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:758)
	at org.testng.TestRunner.run(TestRunner.java:613)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:87)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:886)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:908)
	at java.lang.Thread.run(Thread.java:619)
]]>
    </failure>
  </testcase>
</testsuite>
